pkg/cmd/repo/rename/rename.go

```diff
@@ -104,29 +108,72 @@ func renameRun(opts *RenameOptions) error {
 			"Rename %s to:", ghrepo.FullName(currRepo)), ""); err != nil {
 			return err
 		}
+
+		if !opts.RenameLocalDir {
+			var confirmed bool
+			if confirmed, err = opts.Prompter.Confirm("Would you also like to rename the local directory to the same name?", false); err != nil {
```

Comment:
I haven't been able to get this prompt when running the command `gh repo rename`

Reply:
Sorry for the late reply😥. I've just test `gh repo rename` again, the prompt does show up.